docs/README.md

```diff
@@ -4,18 +4,35 @@
 
 This project provides a .NET tracer that leverages the .NET profiling APIs to support .NET instrumentation and auto-instrumentation without requiring code changes to an application.
 
-## Usage
-
-See [DEVELOPING.md](DEVELOPING.md) for build and running instructions.
-
-See [USAGE.md](USAGE.md) for configuration instructions.
-
 ## Status
 
 This project is in the early stages of development starting with an initial seeding of code from the [.NET Tracer for Datadog APM](https://github.com/DataDog/dd-trace-dotnet). There is a [project board](https://github.com/open-telemetry/opentelemetry-dotnet-instrumentation/projects/1) showing the current work in progress and the backlog.
 
 For more details about the design and roadmap see [DESIGN.md](DESIGN.md).
 
+## Compatibility
+
+### Supported .NET versions
+
+OpenTelemetry .NET Auto-Instrumentation attempts to work with all officially
+supported versions of [.NET Core](https://dotnet.microsoft.com/download/dotnet-core),
+and [.NET Framework](https://dotnet.microsoft.com/download/dotnet-framework)
+except for versions lower than `.NET Framework 4.6.1`.
+
+### Supported OSes
+
+The following operating systems are currently supported:
```

Comment:
When we use the word `supported` here, what are we implying? Are we implying that we have automated tests against these platforms, provide usage instructions and documentation, installation methods, or other things? Should we be more transparent about why we are calling out these operating systems?

Reply:
That is what I had in mind. Not sure if it is worth writing it down.

Reply:
@pellared it seems worth to spell it out. My suggestion would be to change the title to "Compatible OSes" and state that the project is automatically tested against such and such OSes and keep the line about that it should work with other Linux flavors.

Reply:
I tried to make it shorter and more concise